ExportScripts/export_standalone_monolithic.py

```diff
@@ -222,14 +227,31 @@ def process_command(args: list,
 
     #commands are based on 
     #https://github.com/o3de/o3de-multiplayersample/blob/development/Documentation/PackedAssetBuilds.md
-
+    selected_platform = args.platform
+
+    system_platform = platform.system().lower()
+
+    if not selected_platform:
+        logger.info("Platform not specified! Defaulting to Host platform...")
+        if not system_platform:
+            logger.error("Unable to identify host platform! Please supply the platform using '--platform'. Options are [pc, linux, mac].")
+            sys.exit(1)
+        if system_platform == "windows":
+            selected_platform = "pc"
+        elif system_platform == "linux":
+            selected_platform = "linux"
+        elif system_platform == "darwin":
+            selected_platform = "mac"
```

Comment:
For a potential future change, you could put this logic in into a python method and then just have the `default` option for the `--platform` argument just call the function to initialize the platform.
i.e
```python
parser.add_argument('-pl', '--platform', type=str, default=set_default_ap_platform(), choices=['pc', 'linux', 'mac'])
```